app/views/devise/mailer/reset_password_instructions.html.haml

```diff
@@ -0,0 +1,3 @@
+%p= t('devise.mailer.password_change_instructions.headline', {email: @resource.email})
+
+= GitHub::Markdown.render(t('devise.mailer.password_change_instructions.text', {link: link_to t('devise.change_my_password'), edit_password_url(@resource, reset_password_token: @token)}))
```

Comment:
I guess parenthesis are required for `link_to()`, otherwise the hash syntax is invalid.